modules/sfp_twilio.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_twilio
+# Purpose:      Extract data from phone numbers.
+#
+# Author:      Krishnasis Mandal <krishnasis@hotmail.com>
+#
+# Created:     14/06/2020
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+import base64
+import json
+
+class sfp_twilio(SpiderFootPlugin):
+    """Twilio:Footprint,Investigate,Passive:Search Engines:apikey:Obtain data from Twilio from target phone number."""
+    
+    opts = {
+        'account_sid': '',
+        'auth_token': ''
+    }
+
+    optdescs = {
+        'account_sid': 'Twilio Account SID',
+        'auth_token': 'Twilio Auth Token'
+    }
+
+    results = None
+    errorState = False
+
+    def setup(self, sfc, userOpts=dict()):
+        self.sf = sfc
+        self.results = self.tempStorage()
+
+        for opt in list(userOpts.keys()):
+            self.opts[opt] = userOpts[opt]
+
+    def watchedEvents(self):
+        return ["PHONE_NUMBER"]
+
+    # What events this module produces
+    def producedEvents(self):
+        return ["HUMAN_NAME", "COMPANY_NAME", "RAW_RIR_DATA"]
```

Comment:
Module is only reporting HUMAN_NAME and RAW_RIR_DATA, so you can drop COMPANY_NAME.